worlds/soccer.wbt

```diff
@@ -1147,6 +1237,8 @@ Robot {
     Emitter {
```

Comment:
@Adman interesting, this has already been in there?

Reply:
Yes, that's supervisor's emitter.